### PR TITLE
release notes for `edge-21.12.4`

### PR DESCRIPTION
**Pull request by Pothulapati**
## edge-21.12.4

This release adds support for custom HTTP meathods in the viz stats
(i.e CLI and Dashboard). Additionally, It also includes various
other small improvements.

- Added support for custom HTTP methods in the `linkerd-viz` stats
- Updated the health checker to pull trust root from the `linkerd-identity-trust-roots`
  configmap to support cases where they are generated externally (thanks @wim-de-groot)
- Removed unnecessary `installNamespace` bool flag from the
  `linkerd-control-plane` chart (thanks @mikutas)
- Updated the `install` command to error if container runtime check
  fails
- Added support for non-pod endpoints in `GetProfile` responses
- Updated various dependencies across the project (thanks @dependabot)

Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Changes
 
+## edge-21.12.4
+
+This release adds support for custom HTTP meathods in the viz stats
```
Comment (alpeb) [inline CHANGES.md:5]:
```suggestion
This release adds support for custom HTTP methods in the viz stats
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+other smaller improvements.
+
+* Added support for custom HTTP methods in the `linkerd-viz` stats
+* Updated the health checker to pull trust root from the `linkerd-identity-trust-roots`
+  configmap to support cases where they are generated externally (thanks @wim-de-groot)
+* Removed unnecessary `installNamespace` bool flag from the
+  `linkerd-control-plane` chart (thanks @mikutas)
+* Updated the `install` command to error if container runtime check fails
+* Added support for non-pod endpoints in `GetProfile` responses
+* Updated various dependencies across the project (thanks @dependabot)
```
Comment (alpeb) [inline CHANGES.md:16]: c'mon :rofl:

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
 # Changes
 
+## edge-21.12.4
+
+This release adds support for custom HTTP meathods in the viz stats
+(i.e CLI and Dashboard). Additionally, It also includes various
```
Comment (kleimkuhler) [inline CHANGES.md:6]:
```suggestion
(i.e CLI and Dashboard). Additionally, it also includes various
```

File: CHANGES.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+other smaller improvements.
```
Comment (kleimkuhler) [inline CHANGES.md:7]: "various other" is a little redundant; reads better to remove "other".

```suggestion
smaller improvements.
```

**Review by kleimkuhler — APPROVED** [review]

File: CHANGES.md [hunk 5]
```diff
@@ -1,5 +1,20 @@
...
+
+* Added support for custom HTTP methods in the `linkerd-viz` stats
+* Updated the health checker to pull trust root from the `linkerd-identity-trust-roots`
+  configmap to support cases where they are generated externally (thanks @wim-de-groot)
+* Removed unnecessary `installNamespace` bool flag from the
+  `linkerd-control-plane` chart (thanks @mikutas)
+* Updated the `install` command to error if container runtime check fails
+* Added support for non-pod endpoints in `GetProfile` responses
```
Comment (mateiidavid) [inline CHANGES.md:15]: We could perhaps take this out? Not sure how user facing this change is, it's more of an implementation detail 🤔

**Review by mateiidavid — APPROVED** [review]

**Pull request merged**